src/js/components/common/forms/fileupload.test.js

```diff
@@ -57,14 +57,16 @@ describe('FileUpload Component', () => {
     expect(screen.getByText(/test placeholder/i)).toBeInTheDocument();
     // container.querySelector doesn't work in this scenario for some reason -> but querying document seems to work
     const uploadInput = document.querySelector('.dropzone input');
-    await user.upload(uploadInput, menderFile);
+    await act(async () => {
+      await user.upload(uploadInput, menderFile);
+    });
     await waitFor(() => rerender(ui));
 
     expect(uploadInput.files).toHaveLength(1);
     await waitFor(() => expect(document.querySelector('.dropzone input')).not.toBeInTheDocument());
     expect(screen.getByDisplayValue('test.file')).toBeInTheDocument();
 
-    expect(submitMock).toHaveBeenCalled();
-    expect(selectMock).toHaveBeenCalled();
+    await waitFor(() => expect(submitMock).toHaveBeenCalled());
+    await waitFor(() => expect(submitMock).toHaveBeenCalled());
```

Comment:
Shouldn't it be `    await waitFor(() => expect(selectMock).toHaveBeenCalled());` ?